src/example-types/esTwoLevelAggregation.js

```diff
@@ -72,7 +75,17 @@ module.exports = {
                 key: bucket.key || key,
                 doc_count: bucket.doc_count,
               },
-              bucket.twoLevelAgg
+              bucket.twoLevelAgg ||
```

Comment:
If I understand correctly, there isn't any case where `bucket.twoLevelAgg` is a thing anymore, right?

Reply:
There is for cardinality agg. The unit test was failing that is why I added it back.

Reply:
I fixed the unit test.